tests/environment-install.bats

```diff
@@ -0,0 +1,85 @@
+#! /usr/bin/env bats
+# -*- mode: bats; -*-
+# ============================================================================ #
+#
+# Test rust impl of `flox install`
+#
+# ---------------------------------------------------------------------------- #
+
+load test_support.bash
+
+# ---------------------------------------------------------------------------- #
+
+# Helpers for project based tests.
+
+project_setup() {
+  export PROJECT_DIR="${BATS_TEST_TMPDIR?}/test"
+  rm -rf "$PROJECT_DIR"
+  mkdir -p "$PROJECT_DIR"
+  pushd "$PROJECT_DIR" >/dev/null || return
+}
+
+project_teardown() {
+  popd >/dev/null || return
+  rm -rf "${PROJECT_DIR?}"
+  unset PROJECT_DIR
+}
+
+# ---------------------------------------------------------------------------- #
+
+setup() {
+  common_test_setup
+  project_setup
+}
+teardown() {
+  project_teardown
+  common_test_teardown
+}
+
+setup_file() {
+  export FLOX_FEATURES_ENV=rust
+}
+
+# without specifying a name should install to an environment found in the user's current directory.
+@test "i2.a: install outside of shell (option1)" {
+  skip "Environment defaults handled in another phase"
+}
+
+@test "i3: flox install allows -e for explicit environment name;  If .flox does not exist, a .flox is created." {
```

Comment:
This isn't `-e` anymore is it?

Reply:
We're skipping almost all of the tests at this point - we'll update them once we tackle managed environments

Reply:
Got it, but this is replaced with `-n` now right?

Reply:
I think `-n` for init just takes a name, but `-r` for install will take `owner/name`